src/main/java/jenkinsci/plugins/influxdb/generators/JenkinsBasePointGenerator.java

```diff
@@ -104,7 +104,6 @@ public AbstractPoint[] generate() {
                 .addField(BUILD_EXEC_TIME, build.getStartTimeInMillis())
                 .addField(BUILD_MEASURED_TIME, currTime)
                 .addField(BUILD_STATUS_MESSAGE, build.getBuildStatusSummary().message)
-                .addField(BUILD_RESULT, result)
```

Comment:
Please add mentions of this to the [Available metrics](https://github.com/jenkinsci/influxdb-plugin/blob/development/doc/available_metrics.md) and [Breaking changes](https://github.com/jenkinsci/influxdb-plugin/blob/development/doc/breaking_changes.md) documentation. Since this is backwards incompatible, the next release will be a major release (`6.0`). See for example `rf_results` in `available_metrics.md`.